docs/topics/templates/document-attributes.adoc

```diff
@@ -104,16 +119,14 @@ endif::[]
 :LinkAPI: http://windup.github.io/windup/docs/latest/javadoc/
 
 //What about these ones?
-//Links to Jira WINDUP and WINUPRULES project pages:
-:JiraWindupURL: https://issues.redhat.com/projects/WINDUP
-:JiraWindupRulesURL: https://issues.redhat.com/projects/WINDUPRULE
-
-// Point to the GitHub wiki version since there is no product version of this guide.
-:ProductDocCoreGuideURL: https://github.com/windup/windup/wiki/Core-Development-Guide
+//Links to MTA and MTR Jira project pages:
+:JiraMTRURL: https://issues.redhat.com/projects/WINDUP
```

Comment:
Do {JiraMTRURL] AND {JiraMTRURL] appear in the documents? If not, treat :JiraWindupURL:  like the other values (one in the mta ifdef; one in the mtr ifdef). I can check on the ruleset, but for now, i suggest leaving it alone because I think MTA and MTR use the same rules.